scripts/build.py

```diff
@@ -419,7 +421,8 @@ def __init__(self, dynamo_home = None,
         else:
             home = os.environ['HOME']
 
-        self.dynamo_home = dynamo_home if dynamo_home else join(os.getcwd(), 'tmp', 'dynamo_home')
+        self.defold_home = os.path.normpath(join(os.path.dirname(__file__), '..'))
+        self.dynamo_home = dynamo_home if dynamo_home else join(self.defold_home, 'tmp', 'dynamo_home')
```

Comment:
I've been meaning to add this for a long time, in order to more easily author scripts (that are currently based on DYNAMO_HOME, which isn't actually constant!)